src/controllers/v1/example/index.ts

```diff
@@ -48,39 +50,66 @@ export class Example extends Controller {
   }
 
   /**
-   * description
-   * @summary
+   * @summary insert a certificate for initialisation
    */
+  @Example<Certificate.Payload>({
+    id: '52907745-7672-470e-a803-a2f8feb52944',
+    capacity: 10,
+    co2e: 1000,
+  })
   @Post()
   @Response<BadRequest>(400, 'Request was invalid')
   @Response<ValidateError>(422, 'Validation Failed')
   @SuccessResponse('201')
-  public async proposeMatch2(@Body() { example = '' }: example.Request): Promise<example.Response> {
-    this.log.info({ identity: this.identity, example })
+  public async post(@Body() body: Certificate.Request): Promise<Certificate.Response> {
+    this.log.info({ identity: this.identity, body })
 
-    return { message: 'ok' }
+    const formatted = Object.keys(body).reduce(
+      (out, key) => ({
+        [camelToSnake(key)]: body[key],
+        ...out,
+      }),
+      {}
+    )
```

Comment:
move this to a helper please

Reply:
oops, this should have been moved to controller helpers.